network/lib_sddf_lwip/lib_sddf_lwip.mk

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright 2022, UNSW
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+
+ifeq ($(strip $(SDDF_LWIP_NUM_BUFFS)),)
+$(error MAX_NUM_BUFFS must be specified)
+endif
+
+CFLAGS += -DSDDF_LWIP_NUM_BUFFS=$(SDDF_LWIP_NUM_BUFFS)
+
+LIB_SDDF_LWIP_DIR := ${SDDF}/network/lib_sddf_lwip
```

Comment:
Might be better to use location of the Makefile snippet rather than assuming a particular directory hierarchy.